iothub/device/src/DefaultPayloadConvention.cs

```diff
@@ -4,34 +4,32 @@
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    ///
+    /// The default implementation of the <see cref="IPayloadConvention"/> class.
     /// </summary>
+    /// <remarks>
+    /// This class is the default <see cref="IPayloadConvention"/> that will be used for all <see cref="PayloadCollection"/> implementations. This class makes use of the <see cref="JsonContentSerializer"/> serializer and the <see cref="Utf8ContentEncoder"/> unless another implementation is specified for these.
+    /// </remarks>
+    /// <example>
+    /// You can overrride the default instances of both the <see cref="PayloadSerializer"/> and <see cref="PayloadEncoder"/> classes.
```

Comment:
this isn't possible, right? the serializer and encoder only have getters

Reply:
Correct, will change